provisioning/README.md

```diff
@@ -0,0 +1,370 @@
+--data-raw 'PREFIX : <http://example.org/voc#>
+
+SELECT (COUNT(DISTINCT ?x) as ?count)
+WHERE {
+   ?x a :Course .
+}'
+```
+
+```json
+{
+  "head" : {
+    "vars" : [
+      "count"
+    ]
+  },
+  "results" : {
+    "bindings" : [
+      {
+        "count" : {
+          "datatype" : "http://www.w3.org/2001/XMLSchema#integer",
+          "type" : "literal",
+          "value" : "12"
+        }
+      }
+    ]
+  }
+}
+```
+
+Accessing entities using the restricted role/endpoint delivers a smaller count
+
+```console
+curl --location --request POST 'http://localhost:8082/sparql' \
+--header 'Content-Type: application/sparql-query' \
+--header 'Accept: application/json' \
+--data-raw 'PREFIX : <http://example.org/voc#>
+
+SELECT (COUNT(DISTINCT ?x) as ?count)
+WHERE {
+   ?x a :Course .
+}'
+```
+
+```json
+{
+  "head" : {
+    "vars" : [
+      "count"
+    ]
+  },
+  "results" : {
+    "bindings" : [
+      {
+        "count" : {
+          "datatype" : "http://www.w3.org/2001/XMLSchema#integer",
+          "type" : "literal",
+          "value" : "6"
+        }
+      }
+    ]
+  }
+}
+```
+
+### Helm
+
+A helm chart for deploying the remoting agent can be found under [this folder](../charts/provisioning-agent).
+
+It can be added to your umbrella chart.yaml by the following snippet
+
+```console
+dependencies:
+  - name: provisioning-agent
+    repository: https://catenax-ng.github.io/product-knowledge/infrastructure
+    version: 1.9.5-SNAPSHOT
+    alias: my-provider-agent
+```
+
+and then installed using
+
+```console
+helm dependency update
+```
+
+In your values.yml, you configure your specific instance of the remoting agent like this
+
+```console
+#######################################################################################
+# Data Binding Agent
+#######################################################################################
+
+my-provider-agent: 
+  securityContext: *securityContext
+  nameOverride: my-provider-agent
+  fullnameOverride: my-provider-agent
+  resources:
+    requests:
+      cpu: 500m
+      # you should employ 512Mi per endpoint
+      memory: 1Gi
+    limits:
+      cpu: 500m
+      # you should employ 512Mi per endpoint
+      memory: 1Gi
+  bindings: 
+    # disables the default sample binding
+    dtc: null
+    # real production mapping
+    telematics2: 
+      port: 8081
+      path: /t2/(.*)
+      settings: 
+        jdbc.url: 'jdbc:postgresql://intradb:5432/schema'
+        jdbc.user: <path:vaultpath#username>
+        jdbc.password: <path:vaultpath#password>
+        jdbc.driver: 'org.postgresql.Driver'
+      ontology: cx-ontology.xml
+      mapping: |-
+        [PrefixDeclaration]
+        cx-common:          https://w3id.org/catenax/ontology/common#
+        cx-core:            https://w3id.org/catenax/ontology/core#
+        cx-vehicle:         https://w3id.org/catenax/ontology/vehicle#
+        cx-reliability:     https://w3id.org/catenax/ontology/reliability#
```

Comment:
These URL redirect service is pointing to catenax-ng. Do not link/reference to catenax-ng repositories or GH org.

Reply:
The ontology will stay with the association. The redirect services will make sure that once the association decides about the final repo, we can simply move the redirection. Other projects (EDC) use w3id org without havong registered their namespace at all (which is semantically not correct!)